apps/docs/pages/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -62,6 +62,55 @@ Breaking changes are generally only present in major version upgrades of Postgre
 
 If you are upgrading from a significantly older version, you will need to consider the release notes for any intermediary releases as well.
 
+#### Objects dependent on Postgres extensions
+
+pg_upgrade does not support upgrading of databases containing reg\* data types referencing system OIDs.
+If you have created any objects that depend on the following extensions, you will need to recreate them after the upgrade.
+
+Extensions registering system OID-specific reg\* data types:
+
+- `plv8`
+- `plcoffee`
+- `plls`
```

Comment:
Would FDWs using wrappers not be affected?

Reply:
Wrappers were not officially shipped in PG14 (which is the version for which we're dropping them) and we were only trialling them internally until around February.

There was also a break in the wrappers upgrade path before we made them generally available, which is why the drop is executed.